### ISSUE TITLE
Crashing when opened offline

### ISSUE THREAD

**Issue by ChaoticMaximus** [JairusSW/Chess.com#11]
Can the app open offline or is it an emulator of the online version? When I open the app offline it just shows a white screen. I'd love to be able to play bots or whatever if possible. Also, you're the only guy with this windows app that I know of, so kudos to you!

**Comment by JairusSW** [comment]
Yeah, it won't work offline, but what you CAN do, *at least you used to*, is open the page online, navigate to the bots page, and then go offline and play.

**Comment by JairusSW** [comment]
Yeah, it won't work offline, but what you CAN do, *at least you used to*, is open the page online, navigate to the bots page, and then go offline and play.
